tests/models.py

```diff
@@ -204,6 +204,87 @@ def _create_submission(mode: SubmissionModeEnum, complete_question_2: bool = Fal
             _create_submission(SubmissionModeEnum.LIVE, complete_question_2=True)
             _create_submission(SubmissionModeEnum.LIVE, complete_question_2=False)
 
+    @factory.post_generation  # type: ignore
+    def create_completed_submissions_conditional_question_random(  # type: ignore
```

Comment:
I think at some point maybe these just become helper functions rather than part of the definition of the factory itself - they feel super specific. But easy to lift out in the future.

Reply:
Yeah, I agree